test/Microsoft.TestPlatform.Extensions.BlameDataCollector.UnitTests/BlameCollectorTests.cs

```diff
@@ -698,7 +698,7 @@ public void CleanUp()
     private static XmlElement GetDumpConfigurationElement(
         bool isFullDump = false,
         bool collectDumpOnExit = false,
-        bool colectDumpOnHang = false,
+        bool collectDumpOnHang = false,
```

Comment:
Looks like some test is using this parameter explicitely and this is failing the build. Could you look at it?